{{cookiecutter.repo_name}}/.github/workflows/{{cookiecutter._ci_name}}.yaml

```diff
@@ -27,14 +27,16 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       stable-python-version: {{ "${{ steps.get-compatible-python.outputs.stable-python }}" }}
-      python-matrix: {{ '${{ steps.get-compatible-python.outputs.python-versions }}' }}
+      python-matrix: {{ "${{ steps.get-compatible-python.outputs.python-versions }}" }}
     steps:
       - uses: actions/setup-python@v4
         with:
           python-version: "3.11"
 
       - id: get-compatible-python
         uses: MDAnalysis/mdanalysis-compatible-python@main
+        with:
+          release: "latest"
```

Comment:
Right this is my short term fix - updated mdanalysis-compatible-python to have a latest option.

This means that this CI matrix has no way of checking for compatibility with the latest versions of Python...